eclipse-collections-code-generator/src/main/resources/api/block/predicate/primitivePredicate.stg

```diff
@@ -23,8 +23,15 @@ import java.io.Serializable;
  * @since 3.0.
  */
 public interface <name>Predicate
-        extends Serializable
+        extends <if(primitive.specializedStream)>java.util.function.<name>Predicate, <endif>Serializable
 {
+<if(primitive.specializedStream)>
+    default boolean test(<type> value)
+    {
+        return this.accept(value);
+    }
+
+<endif>
```

Comment:
Very minor point, but non-primitive functional interfaces have the new default method after original abstract method. Maybe we want to make the method order consistent?